admin/class-admin.php

```diff
@@ -143,6 +150,15 @@ function register_settings_page() {
 				array( $this, 'load_page' ),
 				null,
 			),
+			array(
+				'wpseo_dashboard',
+				'',
+				__( 'Google Webmaster Tools', 'wordpress-seo' ),
```

Comment:
@jdevalk Should this be changed now? I thought we said it will be still Google Webmaster Tools. What do you think?